Header header
float32 steering_angle            # in Radians, -ve left, +ve right
float32 inst_velocity          # m/sec current speed fast average